content-src/components/TopSites/TopSiteForm.jsx

```diff
@@ -164,23 +164,21 @@ export class TopSiteForm extends React.PureComponent {
       customScreenshotUrl && this.props.previewUrl === this.cleanUrl(customScreenshotUrl);
 
     if (!this.state.showCustomScreenshotForm) {
-      return (<A11yLinkButton onClick={this.onEnableScreenshotUrlForm} className="enable-custom-image-input">
-                <FormattedMessage id="topsites_form_use_image_link" />
-              </A11yLinkButton>);
+      return (<A11yLinkButton onClick={this.onEnableScreenshotUrlForm} className="enable-custom-image-input"
+          data-l10n-id="newtab-topsites-use-image-link" />);
```

Comment:
This is a custom react component, so we'll most likely need to specially handle this prop. It was working before because `<A11yLinkButton>` nests the children.

Reply:
https://github.com/mozilla/activity-stream/blob/baaac1f8301678104e5b364fde465fe1982d56e5/content-src/components/A11yLinkButton/A11yLinkButton.jsx#L10-L11

Reply:
https://github.com/mozilla/activity-stream/blob/baaac1f8301678104e5b364fde465fe1982d56e5/content-src/components/A11yLinkButton/A11yLinkButton.jsx#L10-L11

Oh. I guess it just spreads the `props` which.. I believe we wanted to avoid on plain html elements? @AllegroFox @k88hudson ?

Reply:
Although……… ha. I suppose in this case, fluent-dom will just override the children or any dangerously set html… But safer to not even set in the first place !